runtime/binding-kafka/src/main/java/io/aklivity/zilla/runtime/binding/kafka/internal/cache/KafkaCachePartition.java

```diff
@@ -511,8 +511,8 @@ public void writeEntryFinish(
             if (type.value != null)
             {
                 OctetsFW value = headEntry.value();
-                if (value != null &&
-                    !type.value.validate(value.value(), value.offset(), value.sizeof()))
+                if (value == null ||
+                    !type.value.read(value.value(), value.offset(), value.sizeof()))
```

Comment:
This is saying that `value == null` is a validation failure, but `tombstone` messages have `null` value in Kafka, so log compacted topics would always fail validation.